microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -362,6 +362,24 @@ Local content that's loaded via virtual host name mapping has an HTTP or HTTPS U
 Virtual host name URLs are resolved in WebView2 processes. This is a faster option than `WebResourceRequested`, which resolves in the host app process UI thread.
 
 
+<!-- ---------- -->
+###### Source maps with virtual host name mapping
+
+Source maps are needed to debug the source code of compiled content, such as:
+* Transpiled JavaScript, such as TypeScript or minified JavaScript.
+* Compiled CSS, such as SASS or SCSS.
+
+WebView2 doesn't load source maps that are referenced by content which was loaded by using virtual host name mapping.  
+
+Consider the following example. WebView2 loads JavaScript file `main.js` via virtual host name mapping.  If `main.js` references `main.js.map` as its source map, then `main.js.map` will neither be loaded automatically nor any `WebResourceRequested`<!-- todo: virtual host name mapping?  compare similar sections in this 4 and maybe 3 other files --> event handler will be called to load it.
+
+To use source maps along with virtual host name mapping, choose one of the following approaches:
+
+* Generate inline source maps during compilation of your content. Inline source maps are embedded to the corresponding compiled file.
+
+* Use `WebResourceRequested` event instead, and inline separate source maps to the content at runtime in your `WebResourceRequested` event handler. Use this approach only if your content build system does not support inline source maps.
```

Comment:
```suggestion
* Use `WebResourceRequested` event instead, and inline separate source maps to the content at runtime in your `WebResourceRequested` event handler. Use this approach only if your build system does not support inline source maps.
```
In orig comment, Akhila @ dhveerap asked if the word 'content' really needs to be included, or can omit it.
Can we omit the word "content"?

Reply:
omit 'content'